packages/vue-client/src/components/GameView/SeatComponent.vue

```diff
@@ -15,6 +16,9 @@ defineProps<{
   time_control: IPerPlayerTimeControlBase | null;
   time_config?: ITimeControlConfig;
   is_players_turn: boolean;
+  variant: string;
+  config: unknown;
```

Comment:
Since we're passing in `config` now, should we just get `time_config` from that?

Reply:
Yeah that makes a lot of sense, although now I'm unsure if passing in config is necessary.